generator/routing_generator.cpp

```diff
@@ -1,8 +1,9 @@
 #include "generator/routing_generator.hpp"
 
-#include "generator/borders_generator.hpp"
-#include "generator/borders_loader.hpp"
-#include "generator/gen_mwm_info.hpp"
+#include "borders_generator.hpp"
```

Comment:
Instead of following practices in the edited file you started to introduce your own practices. Look at the top line - it's still "generator/routing_generator.hpp", not "routing_generator.hpp". Please, revert your changes to these lines.